rotkehlchen/inquirer.py

```diff
@@ -483,7 +483,7 @@ def _try_oracle_price_query(
         price, used_main_currency, is_error = ZERO_PRICE, False, True
 
         if from_asset == A_POLYGON_POS_MATIC and ts_now() > POLYGON_POS_POL_HARDFORK:  # after hardfork, we use different oracles  # noqa: E501
-            from_asset = AssetWithOracles('eip155:1/erc20:0x455e53CBB86018Ac2B8092FdCd39d8444aFFC3F6')  # POL token  # noqa: E501
```

Comment:
What does this change do? Doesn't the constructor do a resolution in it?

Reply:
answer: No this is an abstract class. Should never have been initializable/intialized